src/doc/rustdoc/src/unstable-features.md

```diff
@@ -84,6 +84,36 @@ in documentation.
 `#![feature(doc_cfg)]` feature gate. For more information, see [its chapter in the Unstable
 Book][unstable-doc-cfg] and [its tracking issue][issue-doc-cfg].
 
+Something important to be noted is that since [#89596](https://github.com/rust-lang/rust/pull/89596)
+was merged, you don't need to add `#[doc(cfg(..)]` anymore unless you want to override the default
+behaviour. So if we take the previous source code:
+
+```rust
+#![feature(doc_cfg)]
```

Comment:
I believe this behavior will only be enabled if `#![feature(doc_auto_cfg)]` is used.

Reply:
You're absolutely right. Thanks!